.gitattributes

```diff
@@ -0,0 +1 @@
+ins-node/hardware/* linguist-vendored
```

Comment:
This will force GitHub to NOT take into consideration the source files in that folder when determining the repository's language. If not included then "Eagle" will appear as one of the repository's main programming languages, which is not really the case.